flutter_local_notifications/lib/src/platform_specifics/linux/icon.dart

```diff
@@ -23,7 +23,8 @@ class LinuxFileIcon implements LinuxIcon {
 }
 
 /// Represents an icon from binary data
-/// The binary data should be in an image file format which supported by glib, such as common formats like jpg and png
+/// The binary data should be in an image file format which supported by glib, 
+/// such as common formats like jpg and png
 class BytesLinuxIcon implements LinuxIcon {
```

Comment:
This reads slight awkward. Could this perhaps be renamed to something else like `ByteDataLinuxIcon`? Would also make it similar to https://api.flutter.dev/flutter/dart-typed_data/ByteData-class.html